index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
+            My Recent Works
+        </h1>
+
+    </section>
+
+    <section class="section-bg">
+        <div class="bg">
+        </div>
+    </section>
+
+    <section class="multi-post">
+        <h1 class="multipost-content">
+            Multi-Post Stories
+        </h1>
+    </section>
+
+    <div class="paragraph-box">
+        <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
```

Comment:
- [ ] You did amazing job styling the cards but please decrease the space between heading and paragraph following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159077277-e6726bd0-01a0-46c5-b6c7-dfa559dbd184.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159077320-3cac2b82-a167-47c0-9c56-2cbaa4713406.png)